### ISSUE TITLE
Include .wasm files in output trace

### ISSUE THREAD

**Issue by brhx** [vercel/next.js#32612]
Currently it is not possible to import `.wasm` files through webpack on serverless functions and have them stay present in the resulting serverless build, it marks the file as having not been found

![image](https://user-images.githubusercontent.com/8385528/142043415-d846ff84-1ba7-4315-907b-88cdbacac586.png)

**Comment by always-maap** [comment]
yep same issue. workaround mentioned [here](https://github.com/vercel/next.js/issues/29362).

```js
// next.config.js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  webpack(config, { isServer }) {
    if (isServer) {
      config.output.webassemblyModuleFilename = './../static/wasm/[modulehash].wasm';
    } else {
      config.output.webassemblyModuleFilename = 'static/wasm/[modulehash].wasm';
    }
    config.experiments = { asyncWebAssembly: true };
    config.optimization.moduleIds = 'named';

    return config;
  },
};

module.exports = nextConfig;
```

you can check out my [repo](https://github.com/always-maap/Limoo-Lang). building and deploying fine on vercel.

**Comment by kachkaev** [comment]
Thanks @always-maap! I tried your workaround and it fixed `next build`. Do you use wasm in API routes or just pages? My Next API route with WASM crashes, details here: https://github.com/vercel/next.js/issues/29362#issuecomment-1278868452
